Dockerfile

```diff
@@ -3,4 +3,5 @@ FROM python:2.7-onbuild
 RUN apt-get update \
     && apt-get install -y python-protobuf
```

Comment:
you can remove it, I've been removing it from mine during testing. it runs after pip has run and built everything